lib/error.js

```diff
@@ -55,8 +64,8 @@ export const makeError = ({
 	const isError = Object.prototype.toString.call(error) === '[object Error]';
 	const shortMessage = isError ? `${execaMessage}\n${error.message}` : execaMessage;
 	const message = [shortMessage, stdio[2], stdio[1], ...stdio.slice(3)]
+		.map(messagePart => stripFinalNewline(serializeMessagePart(messagePart)))
 		.filter(Boolean)
-		.map(messagePart => serializeMessagePart(messagePart))
 		.join('\n');
```

Comment:
I wonder whether we should separate the error message, stdout and stderr with an empty line or not?

```
Error: this failed

stdout
...

stderr
...
```

vs (current behavior)

```
Error: this failed
stdout
stderr
```

The first separates each part in a clearer way. But the second one is more compact. :thinking: 

Which one do you prefer?

Reply:
The first one

Reply:
Done :+1: